.github/workflows/scripts/start-acs.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/bash
+set -euox pipefail
+
+
+cd $STACKROX_DIR
```

Comment:
@vikin91 IIRC `cd` commands are messing things up. 

Why is changing the directory necessary? IIUC the `test-gh-actions` PR, this is the Github workspace which should be the default working dir of the script anyway.

Reply:
It would be best to avoid changing dirs in the scripts. If that is absolutely necessary, then we should use `pushd` and `popd` in scripts instead of `cd`.

Reply:
After reading the script, I suggest to rewrite it to use absolute paths. This snippet may be then helpful to find the dir in which the script is located (I took it from the `./deploy` scripts):

```bash
DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd)"
```

Reply:
This is needed when running the script locally.

Reply:
Simply using absolute paths instead of changing directories will not work, because of the line `source deploy/common/deploy.sh`. Changing that to `source "${STACKROX_DIR}"/deploy/common/deploy.sh` results in an error 

```
SCANNER_IMAGE="$DEFAULT_IMAGE_REGISTRY/scanner:$(cat "$(git rev-parse --show-toplevel)/SCANNER_VERSION")"
```

since this is not executed in the stackox/stackrox repo it cannot find the SCANNER_VERSION file.

This could be changed to

```
SCANNER_IMAGE="$DEFAULT_IMAGE_REGISTRY/scanner:$(cat "$(git -C $DIR rev-parse --show-toplevel)/SCANNER_VERSION")"
```

where $DIR is the directory of the script.

Reply:
Similar problems occur for start-secured-cluster.sh. I now use absolute paths and pushd instead of cd.